lib/ResourceProviders.ts

```diff
@@ -17,11 +18,17 @@ export class ResourceProviders {
       this.providers[subSet] = [];
     }
 
+    let deps: any[] = Reflect.getMetadata("design:paramtypes", resource)
```

Comment:
Ok. Single quotes in google style.
It seems for me, is not necessary to use <any> for Reflect. I just added `import 'reflect-metadata';` for correct  type definition. What do you think about it?